ffi/loadlib.lua

```diff
@@ -31,6 +36,14 @@ ffi.load = function(lib, global)
     return ffi_load(lib, global)
 end
 
+-- Example usage: `ffi.loadlib("SDL2-2.0", 0, "SDL2-2.0",nil, "SDL2", nil)`,
```

Comment:
If you put an `@module ffi.loadlib` at the top you could also write this as an LDoc function comment. See https://github.com/koreader/koreader/blob/ca569c1355d14ad023eb1bcad7b995ae027e423d/frontend/gettext.lua#L295-L323 for an example.

Reply:
(Or we can just merge it as is, but I find it nice to have; it also shows up in IDEs.)

Reply:
Done.